src/components/admin/AdminCard.tsx

```diff
@@ -12,13 +12,14 @@ import {
   Input,
   EditableInput,
 } from '@chakra-ui/react';
-import { CheckIcon, CloseIcon, EditIcon } from '@chakra-ui/icons';
+import { CheckIcon, CloseIcon, DeleteIcon, EditIcon } from '@chakra-ui/icons';
 import { Assignment, Location } from '@prisma/client';
 import { UseTRPCMutationResult } from '@trpc/react/shared';
 
 interface AdminCardProps {
   assignmentOrLocation: Assignment | Location;
   editMutation: UseTRPCMutationResult<any, any, any, any>;
+  handleDelete: (id: number) => Promise<void>;

```

Comment:
Every component has props that are typed. Try to avoid using `any` but sometimes we might need to.